src/partials/home/hero.js

```diff
@@ -254,7 +254,9 @@ const Hero = ({
             </LinkItem>
           ))}
         </LinkContainer>
-        <LearnMore to="Features">LEARN MORE</LearnMore>
+        <LearnMore to="Features" smooth offset={-25} duration={500}>
```

Comment:
I wonder if we can extract these so that the scrolling is consistent across the app?

Reply:
that's an interesting idea. i'll play with that a bit today. i don't want to over use the animation. Since this link is only occurring on mobile, are you ok with me merging this as is?

Reply:
Yep totally! Go for it 🙂